internal/cli/opsmanager/backup/checkpoints_list.go

```diff
@@ -22,14 +22,19 @@ import (
 	"github.com/mongodb/mongocli/internal/output"
 	"github.com/mongodb/mongocli/internal/store"
 	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/mongodb/mongocli/internal/validate"
 	"github.com/spf13/cobra"
 )
 
+const checkpointsTemplate = `ID	TIMESTAMP{{range .Results}}
+{{.ID}}	{{.Timestamp}}{{end}}
+`
+
 type CheckpointsListOpts struct {
 	cli.GlobalOpts
 	cli.ListOpts
-	clusterName string
-	store       store.CheckpointsLister
+	clusterID string
```

Comment:
👍 Nice to clarify the naming.

Reply:
for context this is a bit of a left over as this method used to work with Atlas, where you need a cluster name